app/src/utils/errors.ts

```diff
@@ -1,35 +1,37 @@
 /* eslint-disable max-classes-per-file */
 export class TooManyRequestsError extends Error {
-  constructor(message) {
+  constructor(message: string | undefined) {
```

Comment:
You can simply use optional parameter:
```suggestion
  constructor(message?: string) {
```